### PR TITLE
PickerItem: Update to correctly use HTMLAttributes vs htmlProps

### PR DESCRIPTION
**Pull request by joschect**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file if publishing 
- [ ] New feature, bugfix, or enhancement
  - [ ] Includes tests
- [ ] Documentation update

#### Description of changes

PickerItem: Update to correctly use HTMLAttributes vs htmlProps

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by micahgodbolt** [comment]
What problem does this solve? Also, 7 commits to change one line? :)

**Comment by joschect** [comment]
More commits = better right? Opposite of golf rules.

This fixes an issue where you might want to pass additional data down through to the picker item, standard information HTML attribute information. Really we shouldn't be using React.Props anywhere.

File: packages/office-ui-fabric-react/src/components/pickers/PickerItem.Props.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-export interface IPickerItemProps<T> extends React.Props<any> {
+export interface IPickerItemProps<T> extends React.HTMLProps<any> {
```
Comment (dzearing) [inline packages/office-ui-fabric-react/src/components/pickers/PickerItem.Props.ts]: What? I thought you'd use HTMLAttributes... and not "any"

Comment (joschect) [inline packages/office-ui-fabric-react/src/components/pickers/PickerItem.Props.ts]: Whoops, you're right.

Comment (joschect) [inline packages/office-ui-fabric-react/src/components/pickers/PickerItem.Props.ts]: okay fixed it.

**Pull request merged**